<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora √Ålgebra LIneal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_calcular">
          <property name="text">
           <string>Calcular</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="4" column="0" colspan="2">
            <widget class="QTextEdit" name="textEdit_input">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Resultaddo</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Entrada</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTextBrowser" name="textBrowser_result"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Matrices</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <widget class="QSpinBox" name="spinBox_matrices_numeromatrices">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QFrame" name="frame_matrices_matrizc">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Matriz C</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QTextEdit" name="textEdit_mc"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QFrame" name="frame_matrices_matrizb">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Matriz B</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="1">
               <widget class="QTextEdit" name="textEdit_mb"/>
              </item>
              <item row="0" column="0">
               <widget class="QComboBox" name="comboBox_matrix_ab_op">
                <item>
                 <property name="text">
                  <string>--Select--</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>*</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>+</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>-</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QFrame" name="frame_matrices_matrizd">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Matriz D</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QTextEdit" name="textEdit_md"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Matriz A</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit_ma"/>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="comboBox_matrix_a_op">
             <item>
              <property name="text">
               <string>--Select--</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Traspuesta</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Inversa</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Determinante</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="3" rowspan="3">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Resultado</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser_matrix_result"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_matrices_calcular">
          <property name="text">
           <string>Calcular</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsdfsadf">
   <property name="text">
    <string>sdfsadf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
